website_cookie_notice/models/res_company.py

```diff
@@ -62,7 +46,7 @@ class Company(models.Model):
         string="cookieResetButton")
     cookieWhatAreTheyLink = fields.Char(
         string="cookieWhatAreTheyLink",
-        default="http://www.allaboutcookies.org/")
+        default="//wikipedia.org/wiki/HTTP_cookie")
```

Comment:
This is called a protocol relative URI, which will adapt to the protocol you are viewing the site on. I didn't want to force HTTPS on non-HTTPS sites, and adding `http:` will yield the certificate error that was reported in the issue